pkg/shared/globalconfig.go

```diff
@@ -75,6 +75,19 @@ func GetDefaultServiceType() string {
 	return DefaultServiceType
 }
 
+// GetDefaultSvcClusterDomainBase extracts the default svc cluster domain
+// from the globalConfig CR data if present, otherwise returns the default
+// value (NodePort).
+func GetDefaultSvcClusterDomainBase() string {
```

Comment:
Shouldn't the name of this method should be GetSvcClusterDomainBase() since it doesn't always return the default?

Reply:
Yep agreed. That's my only remaining "issue" on this one, looks good otherwise.